SpatialGDK/Source/SpatialGDK/Public/Utils/InspectionColours.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Containers/UnrealString.h"
+#include "Math/Color.h"
```

Comment:
Unreal spells it color such as `FColor` that is used below but you type colour in the file name and new function names. I think it's better to be consistent (than British :)) and I think there's something in the coding guideline about spelling things the American way...

Reply:
Also this is from our existing code but with different spelling - https://github.com/spatialos/UnrealGDK/pull/1581/files#diff-c0e52405e67491a66b7d0bb393843fd3R363

Reply:
Boo america